superset/assets/stylesheets/superset.less

```diff
@@ -51,7 +51,8 @@ input.form-control {
 
 .slice_description{
     padding: 8px;
-    margin: 5px;
+    margin-top: 5px;
```

Comment:
`margin: 5px 0` will do the same

Reply:
Thanks for the recomendation, I forgot that I could like that too.